superset/config.py

```diff
@@ -459,7 +459,6 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
     # Allow users to export full CSV of table viz type.
     # This could cause the server to run out of memory or compute.
     "ALLOW_FULL_CSV_EXPORT": False,
-    "UX_BETA": False,
```

Comment:
Note for future feature flag removals: we should always try to enable FFs by default a good time in advance of removal.

Reply:
Agreed 👍🏼